compiler/template/native/testdata/step/loop_map/want.yml

```diff
@@ -5,15 +5,15 @@ steps:
     image: golang:latest
     pull: true
     ruleset:
-      event: [ push, pull_request ]
+      event: [ push, pull_request, delete ]
 
   - name: sample_test_latest
     commands:
       - go test ./...
     image: golang:latest
     pull: true
     ruleset:
-      event: [ push, pull_request ]   
+      event: [ push, pull_request, delete ]   
```

Comment:
Is there any reason why all these test pipelines are including `delete`?

Reply:
im not sure, I didn't add that and I'm a bit unsure as to why all these files are included in my PR